storybook/src/components/accordion/accordion.css-template.ts

```diff
@@ -27,6 +27,9 @@ export const cssAccordion: ComponentImplementation<Accordion<TemplateResult>> =
             ? html`<span class="dso-icon">${iconTemplate({ icon: section.icon })}</span>`
             : nothing}
           ${section.handleTitle}
+          ${section.badgeMessage
+            ? html`${badgeTemplate({ status: section.badgeStatus, message: section.badgeMessage })}`
```

Comment:
Je kan ervoor kiezen om direct `badgeTemplate()` aan te roepen:

```suggestion
            ? badgeTemplate({ status: section.badgeStatus, message: section.badgeMessage })
```

Reply:
Gedaan.